.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install --force
+          && go get -u ./...
+          && git checkout {{ event.head.sha }} 
+          && go test -v -race ./...
+          && gometalinter --vendor --disable-all --enable=gotype --enable=golint 
```

Comment:
But we might have to run `go install ./...` before this. _gotype_ uses the compiled data that is generated by `install` to run its checks.